features/bevragen/overzicht.md

```diff
@@ -59,6 +59,27 @@ De volgende velden hoeven niet met de fields parameter te worden gevraagd. Deze
 - opschortingBijhouding
 - rni
 
+### Filteren van verblijfplaats velden en de adresregel velden van de adressering gegevensgroep
+
+Voor verblijfplaats en adressering zijn er twee autorisatie profielen:
+
+- geautoriseerd voor verblijfplaats binnenland en verblijfplaats buitenland gegevens
+- geautoriseerd voor alleen verblijfplaats binnenland gegevens
+
+Consumers die geautoriseerd zijn voor alleen verblijfplaats binnenland gegevens kunnen hierdoor de standaard veld paden van verblijfplaats en adresregels niet gebruiken om alleen verblijfplaats binnenland velden te vragen. Met deze veld paden worden zowel verblijfplaats binnenland als verblijfplaats buitenland gevraagd.
+Om het mogelijk te maken voor consumers die niet geautoriseerd zijn voor het bevragen van verblijfplaats buitenland gegevens, zijn de volgende twee fields aliassen gedefinieerd waarmee kan worden aangegeven dat alleen verblijfplaats binnenland gegevens wordt opgevraagd: 
+
+- verblijfplaatsBinnenland
+- adresseringBinnenland
+
```

Comment:
Leg hier uit welke gegevens je met 'verblijfplaatsBinnenland' en 'adresseringBinnenland' op kunt halen zodat duidelijk is dat je hiermee zowel 'Adres' en 'Locatie' terug kunt krijgen.

Reply:
Dat heb ik volgens mij gedaan (misschien niet uitputtend) in de feature files. Maar geef anders een voorstel hoe jij het zou doen